packages/app/src/cli/services/function/common.ts

```diff
@@ -50,3 +51,12 @@ export async function inFunctionContext({
     throw new AbortError('Run this command from a function directory or use `--path` to specify a function directory.')
   }
 }
+
+export async function readSchemaFile(directory: string): Promise<string | undefined> {
```

Comment:
This isn't really reading the file, it's just returning the schema path of the function by concatenating the given path. I think we can make this harder to misuse by ensuring it's operating on a function. What if we did something like this instead?

```typescript
export async function schemaPath(extension: ExtensionInstance<FunctionConfigType>): Promise<string | null> {
  const path = joinPath(extension.directory, 'schema.graphql');
  if (await fileExists(path)) {
    return path
  }

  return null
}
```